### ISSUE TITLE
`matrix` isn't an allowed scheme in message html

### ISSUE THREAD

**Issue by tezlm** [matrix-org/matrix-spec#1640]
https://spec.matrix.org/v1.8/client-server-api/#mroommessage-msgtypes

> a | name, target, href (provided the value is not relative and has a scheme matching one of: https, http, ftp, mailto, magnet)

Maybe `matrix:` didn't exist when this part of the spec was written?

**Comment by MTRNord** [comment]
This starts to cause fallout on details of gematik TIM Spec. Since gematik demands 1.3 spec (which is weird enough) MVPs are built without support of this. It seems rather weird that the gematik spec both requires and due to this spec but also not requires the use of it. (They require it downlevel in vCards. But then as they require 1.3 users will not be allowed to use them in chat.)

**Comment by zecakeh** [comment]
Isn't this a duplicate of #1108?

**Comment by KitsuneRal** [comment]
Yes it is; but opinions in SCT seem to be split on whether this is needs an MSC. In any case, let's continue there.
